src/app/pages/details/ally/ally.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="text">
     <div class="logo"></div>
-    <h2>{{name}}</h2>
+    <h4>{{name}}</h4>
```

Comment:
Having non-contiguous h-levels may be an accessibility issue.
http://stackoverflow.com/questions/8778829/is-it-ok-to-skip-heading-levels